libpod/runtime_ctr.go

```diff
@@ -96,6 +96,13 @@ func (r *Runtime) NewContainer(rSpec *spec.Spec, options ...CtrCreateOption) (c
 			return nil, err
 		}
 	}
+
+	if ctr.config.ImageVolumes {
```

Comment:
This would be better in Init() than NewContainer()

Reply:
fixed